QuizMakerAngular/src/app/services/user.service.ts

```diff
@@ -136,9 +136,9 @@ export class UserService {
     return this.http.post(this.link + '/removeTeamMember', data, this.httpOptions).toPromise();
   }
 
-  getUserDashboard(userId: number): Promise<User[]> {
-    let url = `${this.link}/getTeamMembers/${userId}`;
-    return this.http.get<User[]>(url, this.httpOptions).toPromise();
+  getUserDashboard(userId: string): Promise<User[]> {
+    let url = `${this.link}/dashboard/${userId}`;
+    return this.http.get<any>(url, this.httpOptions).toPromise();
   }
```

Comment:
Why this change?

Reply:
It doesnt actually return just user stuff. it resturns a bunch of stuff for the quiz

Reply:
This is how we get the user's quiz list. Was it being used elsewhere?